apis/system/tests/write_r_read_python.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+import tiledbsoma as soma
+
+from .common import TestReadPythonWriteR
+
+
+class TestDataframeWriteRReadPython(TestReadPythonWriteR):
+    @pytest.fixture(scope="class")
+    def R_dataframe(self):
+        base_script = f"""
+        require("tiledbsoma")
+        require("arrow")
+
+        df_schema <- schema(
+        field("foo", int32()),
+        field("bar", float64()),
+        field("baz", string()),
+        field("quux", bool())
+        )
+
+        platform_config <- NULL
+        tiledbsoma_ctx <- NULL
+        sdf <- SOMADataFrameCreate("{self.uri}", df_schema, c("foo"), platform_config, tiledbsoma_ctx)
```

Comment:
Because NULL is the default argyment these three lines can be written just one shorter line:

```suggestion
        sdf <- SOMADataFrameCreate("{self.uri}", df_schema, "foo")
```